### PR TITLE
Add extended ACL commands for container

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

File: container.go [hunk 1]
```diff
@@ -329,3 +346,104 @@ func listContainers(c *cli.Context) error {
...
+		sig   []byte
+		key   = getKey(c)
+		host  = getHost(c)
+		conn  *grpc.ClientConn
+		sCID  = c.String(cidFlag)
+		sEACL = c.String(eaclFlag)
+		ctx   = gracefulContext()
+	)
+
+	if sCID == "" || sEACL == "" {
```
Comment (AlexVanin) [inline container.go:364]: Add `empty` preset support. 

```
neofs-cli set-eacl --cid <...> --eacl empty
```

This command should set `[]byte{}` to the extended ACL field in `SetExtendedACLRequest` message.

Comment (cthulhu-rider) [reply container.go]: Done.

File: object.go [hunk 2]
```diff
@@ -147,10 +153,41 @@ var (
...
+	}
+
+	bearer := new(service.BearerTokenMsg)
+	bearer.SetExpirationEpoch(math.MaxUint64)
+	bearer.SetACLRules(bearerRules)
+	bearer.SetOwnerID(owner)
+
+	req.SetBearer(bearer)
+
+	fmt.Println("ADADADA")
```
Comment (AlexVanin) [inline object.go:186]: Remove debug line

File: go.mod [hunk 3]
```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/mr-tron/base58 v1.2.0
-	github.com/nspcc-dev/neofs-api-go v1.0.0
+	github.com/nspcc-dev/neofs-api-go v1.0.1-0.20200618132644-03bc5c5f895f
```
Comment (AlexVanin) [inline go.mod:7]: Change to `v1.1.0`

Comment (cthulhu-rider) [reply go.mod]: Rdy.

**Pull request merged**